src/interface.js

```diff
@@ -0,0 +1,76 @@
+$(document).ready(function() {
+  var bowlingFrame = new BowlingFrame();
+  var game = new Game(bowlingFrame);
+  var frameNumber = 1;
+  //
+  // $('#bowl-button').click(function() {
+  //   if (game.currentFrame.rollNumber === 2) {
+  //     var $score = game.bowl(game.currentFrame);
+  //     $('#scoreboard').append(
+  //       `<tr id="${frameNumber}"><td>${frameNumber}</td><td>${
+  //         game.currentFrame.rollNumber
+  //       }</td><td>${$score}</td><td>${game.scoreboard.score(
+  //         game.currentFrame
+  //       )}</td></tr>`
+  //     );
+  //     frameNumber += 1;
+  //     game.createNewFrame();
+  //   } else {
+  //     console.log('first');
+  //     var $score = game.bowl(game.currentFrame);
+  //     $('#scoreboard').append(
+  //       `<tr id="${frameNumber}"><td>${frameNumber}</td><td>${game.currentFrame
+  //         .rollNumber - 1}</td><td>${$score}</td></tr>`
+  //     );
+  //   }
+  // });
+
+  $('.buttons')
```

Comment:
I would definitely agree with you that this could do with some refactoring and breaking out into separate functions.  It's good that you identified this as something you would like to have done, and I'm sure given more time you would have been able to.